hera_mc/cm_handling.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, hpn, rev, at_date):
         self.connections = None  # This is the PartConnectionDossierEntry class
         self.geo = None  # This is the geo_location.GeoLocation class
 
-    def __repr__(self):
+    def __repr__(self):  # pragma: no cover
```

Comment:
this should be coverable. There's an example starting on line 35 of `test_server_status.py`.